chrome/Echofon/content/overlay.js

```diff
@@ -868,8 +868,8 @@ var EchofonOverlay = {
 
   onFinishOAuth: function(user_id) {
     var account = EchofonAccountManager.instance().get(user_id);
-    if (EchofonCommon.pref().getIntPref("activeUserId") == 0) {
-      EchofonCommon.pref().setIntPref("activeUserId", account.user_id);
+    if (EchofonCommon.pref().getCharPref("activeUserIdStr") == 0) {
```

Comment:
Yes, maybe simplify boolean checks of this kind
`getBoolPref("foo") == false` => `!getBoolPref("foo")`
`getCharPref("bar") == ''` => `!getCharPref("bar")`
Also it is a bit more explicit about default values when testing on them?
`getIntPref("baz") == 0` / `getIntPref("baz") > 0`